.github/workflows/ci.yaml

```diff
@@ -208,97 +204,93 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        PYTHON_VERSION: [ '3.8' ]
-        SA_VERSION: ["<2.0"]
+        env:
+          - snowflake-py38
+          - snowflake-py39
+          - snowflake-py310
+          - snowflake-py311
+          - snowflake-py312
+          - snowflake-sa1
+          - snowflake-sa2
```

Comment:
We werent testing sqlalchemy 2. Although It might be interesting to give it a second try.

Furthermore, we weren't testing all python versions for these more "esoteric" databases to reduce time, and costs.